lib/events/test/suite.go

```diff
@@ -206,14 +217,16 @@ Outer:
 
 // SessionEventsCRUD covers session events
 func (s *EventsSuite) SessionEventsCRUD(t *testing.T) {
+	loginTime := s.Clock.Now().UTC()
```

Comment:
in SessionEventsCRUD, I have moved timestamps used for events generation to separate variables and make sure that login and session.start are not equal. Dynamo is using fakeClock in it's tests (I have no idea why, in integration it should use real clock) and both events had the same timestamp due to usage of fakeclock. It results in race condition on searching events, we coudln't be sure which one will be first, login or session.start. 
This race existed before, from https://github.com/gravitational/teleport/pull/22071/commits/7e7b0550139a3642ecfd5e4b7b1e005bdf475bf8